lib/puppet/ssl/oids.rb

```diff
@@ -84,7 +84,7 @@ def self.register_puppet_oids()
   #  '1.3.6.1.4.1.34380.1.2.1.2':
   #    shortname: 'myothershortname'
   #    longname: 'Other Long name'
-  def self.load_custom_oid_file(custom_oid_file, map_key='oid_mapping')
+  def self.parse_custom_oid_file(custom_oid_file, map_key='oid_mapping')
```

Comment:
Well, this breaks API, the method is neither Ruby private nor API @private. The original method name must be retained for semver reasons.
Puppet can use the new method name, and the old can delegate to the new after having issued a deprecation warning.

In new code always add @api private to classes, or all methods to prevent not being allowed to make changes. It is allowed to make methods @api private without breaking semver.